src/Domain/Model/Movie/Movie.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Fusani\Omdb\Domain\Model\Movie;
+
+class Movie
```

Comment:
This is a pretty simple model but I imagine we'll add more to it later.

Also, movie implies feature film and this will include tv shows as well so we should consider a different name here.